apps/app/locals.tf

```diff
@@ -13,6 +13,31 @@
 #  limitations under the License.
 
 locals {
+  repository_name = var.app_id
+
   feature_store_odbc = "Driver={ODBC Driver 18 for SQL Server};Server=tcp:${data.azurerm_mssql_server.feature_store.fully_qualified_domain_name},1433;Database=${var.feature_store_db_name};Authentication=ActiveDirectoryMsi;Encrypt=yes;TrustServerCertificate=no;Connection Timeout=30;"
-  is_prod            = length(regexall("prod", lower(var.environment))) > 0
+  acr_repository     = var.app_id
+  create_repo        = var.app_config.managed_repo != null
+
+  staging_slot_name   = "staging"
+  core_gh_env         = var.environment
+  core_branch_name    = local.core_gh_env
+  staging_gh_env      = var.app_config.add_staging_slot ? "${var.environment}-testing_slot" : null
```

Comment:
`nit` maybe we should change `add_staging_slot` to `add_testing_slot` as that's in line with the suffix we're adding?

Reply:
yeah.. should probably do a blanket find+replace staging->testing